docs/vm-installation.md

```diff
@@ -0,0 +1,115 @@
+# Installing antrea-agent on VM
+
+`antrea-agent` can be run on Linux/Windows VMs. Below are the steps needed to configure and run
+antrea-agent on VMs
+
+## Prerequisites on Kubernetes cluster
+
+1. Enable ExternalNode feature on the `antrea-controller`
+2. Create a Namespace for `antrea-agent`
+
+```bash
+kubectl create ns vm-ns
+```
+
+3. Create a ServiceAccount, ClusterRole and ClusterRoleBinding for `antrea-agent`
```

Comment:
Could we also briefly talk about what changes are needed to use a different Namespace other than `vm-ns`?